src/nvim/ops.c

```diff
@@ -1169,8 +1162,7 @@ static void stuffescaped(char_u *arg, int literally)
  * If "regname" is a special register, return TRUE and store a pointer to its
  * value in "argp".
  */
-int 
-get_spec_reg (
+int get_spec_reg(
```

Comment:
In future PRs please avoid style changes in functions that otherwise are unrelated to the subject of the PR, or at least in a separate commit so that semantic changes (integer types and casts in this case) can be reviewed separately from the stylistic ones. (small changes close to lines that are changed anyway are of course ok, and sometimes required by the linter)